account_mass_reconcile_as_job/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+{
+    'name': 'Account Mass Reconcile as Jobs',
+    'version': '12.0.0.1.0',
+    'category': 'Accounting',
+    'depends': [
+        'connector',
```

Comment:
you don't need `connector`, only `queue_job`
```suggestion
        'queue_job',
```

(`queue_job` was extracted from `connector` in version 10.0)